### DISCUSSION TITLE
Mislead in score multiplier colours

### DISCUSSION THREAD

**Discussion by FluffyPunk** [ppy/osu#28924] [Ideas] [answered]
I think that it would be more appropriate to make score multipliers opposite to current.
It's okay when you make it red for some diff values, such as it is in stable when you drop HR, but for score it will be more accurate to make it green instead of red. Same about <1 value, red instead of green, as mark of score penalty for easier mods
![image](https://github.com/user-attachments/assets/bd022b0b-50fe-4d9f-b863-7bcac7c1a76f)

**Comment by SupDos** [comment, marked as answer]
It was changed on purpose to match the colors in other places (the mod icons themselves, the number details on the mod select screen) https://github.com/ppy/osu/pull/28620
